lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -24,10 +25,26 @@ def execute(self) -> int:
             self.config,
             self.logger,
         )
-        crt.copy_result_tb(
+        res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        return 0
+
+        try:
+            msg = res.json()["message"]
+        except requests.exceptions.JSONDecodeError:
+            msg = "FOO"
```

Comment:
I think we should be checking `res.OK` (or similar) before we try to JSON-ize the response.  E.g., if everything is "OK", then we don't need to do anything with the message.  (Although, we might want to handle `200` differently from `201`.)

Reply:
I don't think so: the response will contain HTTP errors (e.g. the 413/Entity too large) which are not OK in this sense. I did change the handling as Dave and you suggested.

Reply:
The server actually always generates a `json["message"]` response payload for upload, so functionally it's fine to process this unconditionally. On 201 it's "File successfully uploaded" (which we probably don't want to use), and for 200 it's "Dataset already exists".